documentation/images/plantuml/boardfarm_prplMesh_classes.puml

```diff
@@ -0,0 +1,165 @@
+@startuml
+class LinuxDevice {
+    string tftp_dir
```

Comment:
I think the UML file is there to convey a message, not to have a one-to-one description of the code - the code is better at doing that :-). So, I think only the members that are really interesting (i.e. will be used in our tests) should be mentioned here.

Reply:
Fixed in update! I have left only crucial fields and methods in boardfarm classes.